fvm/evm/emulator/state/base.go

```diff
@@ -21,6 +21,8 @@ const (
 	CodesStorageIDKey = "CodesStorageIDKey"
 )
 
+var emptyHash = gethCommon.Hash{}
```

Comment:
Do we want to export `emptyHash` since it is used as sentinel value to indicate end of iteration?

Reply:
good idea.